pkg/segment/tracing/handler/tracehandler.go

```diff
@@ -621,7 +627,47 @@ func ProcessDependencyRequest(ctx *fasthttp.RequestCtx, myid uint64) {
 		return
 	}
 	ctx.SetStatusCode(fasthttp.StatusOK)
+}
+func ProcessDepgraphRequest(ctx *fasthttp.RequestCtx, myid uint64) {
```

Comment:
nit: can you make the names `ProcessDepgraphRequest` and `ProcessDependencyRequest` more distinguishable, or add a comment above them to specify what they do? Right now the name doesn't clarify that `ProcessDependencyRequest` aggregates already computed graphs while `ProcessDepgraphRequest` builds it from scratch

Reply:
done, i added comments and updated the method names as well